.github/workflows/weekly.yml

```diff
@@ -21,7 +21,7 @@ jobs:
             SKIP_ALGS: 'SPHINCS\+-SHA(.)*s-simple,SPHINCS\+-SHAKE-(.)*,Classic-McEliece-[^3](.)*'
           - name: extensions
             container: openquantumsafe/ci-ubuntu-focal-x86_64:latest
-            CMAKE_ARGS: -DOQS_DIST_BUILD=OFF -DOQS_OPT_TARGET=haswell -DCMAKE_BUILD_TYPE=Debug -DOQS_ENABLE_TEST_CONSTANT_TIME=ON
+            CMAKE_ARGS: -DOQS_DIST_BUILD=OFF -DOQS_OPT_TARGET=auto -DCMAKE_BUILD_TYPE=Debug -DOQS_ENABLE_TEST_CONSTANT_TIME=ON
```

Comment:
If I'm not mistaken the target 'haswell' was chosen because [Valgrind](https://bugs.kde.org/show_bug.cgi?id=383010) which is used for constant-time testing doesn't support AVX512 yet. The run may fail on newer CPU generations. Perhaps better keep the haswell target.

Reply:
My bad, it looks like this belongs to a commit from main unrelated to CROSS. I just rebased and force-pushed.

Reply:
Thanks for rebasing.